src/form_handler.js

```diff
@@ -0,0 +1,13 @@
+/*
+  eslint-disable no-unused-vars, no-alert, no-mixed-operators, import/prefer-default-export
+*/
+
+const handleForm = (formId) => {
+  const form = document.getElementById(formId);
+  if (form.elements.namedItem('city').value) {
+    return form.elements.namedItem('city').value;
+  }
+  return null;
+};
+
+export { handleForm };
```

Comment:
export default works here too

Also use the rules that are necessary in each file, all the commented out rules here do not look like an issue for this file at all, once you correct the export default